compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/test/conv2d_to_im2col.mlir

```diff
@@ -1,16 +1,16 @@
-// RUN: iree-opt --split-input-file --pass-pipeline="builtin.module(util.func(iree-linalg-ext-convert-conv2d-to-im2col-op))" %s | FileCheck %s
```

Comment:
Why change everything to `func.func` here?

Reply:
Oh, I forgot to change this back.